packages/web/src/lib/myalgowallet-mode.ts

```diff
@@ -106,12 +113,24 @@ export class MyAlgoWalletSession {
 	 * @returns array of raw signed txns | null. null representes that the txn in array is NOT signed
 	 * by wallet user (i.e signable by someone else).
 	 */
-	async signTransactionGroup(txns: TransactionInGroup[], 
-		signOptions?: SignTransactionOptions): Promise<SignedTx[]> {
-		const txnsGroup = txns.map((v) => v.txn);
+	async signTransactionGroup(
+		txns: TransactionInGroup[],
+		signOptions?: SignTransactionOptions
+	): Promise<SignedTx[]> {
+		let txnsGroup = txns.map((v) => v.txn);
 		const groupID = algosdk.computeGroupID(txnsGroup);
-		for (let i = 0; i < txns.length; i++) txnsGroup[i].group = groupID;
-		return await this.connector.signTransaction(txnsGroup.map((txn) => txn.toByte()), signOptions);
+		for (let i = 0; i < txns.length; i++) {
+			// called from executeTx where groupID is already assigned
+			if (txnsGroup[i].group) {
```

Comment:
You don't need continue here. Just use not operator here (!txnsGroup[i].group) and else code will be moved inside if block.